test/Faker/Provider/es_UY/PersonTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Faker\Test\Provider\es_UY;
+
+use Faker\Provider\es_UY\Person;
+use Faker\Test\TestCase;
+
+/**
+ * @group legacy
+ */
+final class PersonTest extends TestCase
+{
+    public function testCI()
+    {
+        //Check validation
+        self::assertTrue($this->validateCi('57351030'));
+        self::assertTrue($this->validateCi('57351046'));
+        self::assertFalse($this->validateCi('5735103-3'));
+        self::assertFalse($this->validateCi('57351033'));
+
+        //Check generation
+        self::assertTrue($this->validateCi($this->faker->ci));
```

Comment:
Testing this twice does not make it more reliable. Either test it once with a seed or in a for loop.

Reply:
Ok.